filament/src/details/Allocators.h

```diff
@@ -19,18 +19,33 @@
 
 #include <utils/Allocator.h>
 
+#ifndef FILAMENT_PER_RENDER_PASS_ARENA_SIZE_IN_MB
+#  define FILAMENT_PER_RENDER_PASS_ARENA_SIZE_IN_MB 2
```

Comment:
Our indentation style is 4 spaces, not 2

Reply:
Will fix.